bootstrap/src/test/java/org/shareio/backend/infrastructure/controller/DeleteRESTControllerIT.java

```diff
@@ -0,0 +1,65 @@
+package org.shareio.backend.infrastructure.controller;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.shareio.backend.core.model.vo.AccountType;
+import org.shareio.backend.infrastructure.dbadapter.entities.AddressEntity;
+import org.shareio.backend.infrastructure.dbadapter.entities.SecurityEntity;
+import org.shareio.backend.infrastructure.dbadapter.entities.UserEntity;
+import org.shareio.backend.infrastructure.dbadapter.repositories.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@AutoConfigureMockMvc
+public class DeleteRESTControllerIT {
+
+    @Autowired
+    UserRepository userRepository;
+
+    @Autowired
+    private MockMvc mvc;
+
+    private UserEntity generateUserEntity(UUID userId){
+        return new UserEntity(null, userId, "jan.kowalski@poczta.pl", "Jan", "Kowal",
+                LocalDateTime.of(2000, 12, 31, 12, 0, 0),
+                UUID.randomUUID(),
```

Comment:
Suggestion: tests should be fully deterministic. Maybe we should use UUIDs from string instead of random?